test/Api.Test/SecretsManager/Controllers/RequestSMAccessControllerTests.cs

```diff
@@ -0,0 +1,82 @@
+﻿#nullable enable
+using System.Security.Claims;
+using Bit.Api.SecretsManager.Controllers;
+using Bit.Api.SecretsManager.Models.Request;
+using Bit.Core.Context;
+using Bit.Core.Entities;
+using Bit.Core.Exceptions;
+using Bit.Core.Models.Data.Organizations.OrganizationUsers;
+using Bit.Core.Repositories;
+using Bit.Core.SecretsManager.Commands.Requests.Interfaces;
+using Bit.Core.Services;
+using Bit.Test.Common.AutoFixture;
+using Bit.Test.Common.AutoFixture.Attributes;
+using NSubstitute;
+using NSubstitute.ReturnsExtensions;
+using Xunit;
+
+namespace Bit.Api.Test.SecretsManager.Controllers;
+
+[ControllerCustomize(typeof(RequestSMAccessController))]
+[SutProviderCustomize]
+public class RequestSMAccessControllerTests
+{
+    [Theory]
+    [BitAutoData]
+    public async Task RequestSMAccessFromAdminst_WhenSendingNoModel_ShouldThrowNotFoundException(
+    User user, SutProvider<RequestSMAccessController> sutProvider)
+    {
+        // Arrange
```

Comment:
⛏️  Can we remove the `// Arrange` and `// Act & Assert` comments? 

For the SM test code base, we don't have them anywhere else.